src/docs/Glossary.adoc

```diff
@@ -90,22 +94,74 @@ of a <<_glossary_workflow,*Workflow*>>.
+
+[[_glossary_task_executable_script]]
+Script Task::
+  A <<_glossary_task_executable,*Task Executable*>> defined as a script execution.
+
+[[_glossary_task_executable_java]]
+Java Task::
+  A <<_glossary_task_executable,*Task Executable*>> defined as a Java class execution.
+
+[[_glossary_task_executable_native]]
+Native Task::
+  A <<_glossary_task_executable,*Task Executable*>> defined as a native command execution.
+
+[[_glossary_additional_task_scripts]]
+Additional Task Scripts::
+   A collection of scripts part of a <<_glossary_task,*ProActive Task*>> definition which can be used in complement to the main <<_glossary_task_executable,*Task Executable*>>. Additional Task scripts can either be <<_glossary_selection_script,*Selection Script*>>, <<_glossary_fork_environment_script,*Fork Environment Script*>>, <<_glossary_pre_script,*Pre Script*>>, <<_glossary_post_script,*Post Script*>>, <<_glossary_control_flow_script,*Control Flow Script*>> or <<_glossary_clean_script,*Cleaning Script*>>
+
+[[_glossary_selection_script]]
+Selection Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and used to select a specific <<_glossary_proactive_node,*ProActive Node*>> to execute a <<_glossary_task,*ProActive Task*>>.
+
+[[_glossary_fork_environment_script]]
+Fork-Environment Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run on the <<_glossary_proactive_node,*ProActive Node*>> selected to execute the Task to configure the forked Java Virtual Machine to execute the task.
+
+[[_glossary_pre_script]]
+Pre Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run inside the forked Java Virtual Machine, before the <<_glossary_task_executable,*Task Executable*>>.
+
+[[_glossary_post_script]]
+Post Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run inside the forked Java Virtual Machine, after the <<_glossary_task_executable,*Task Executable*>>
+
+[[_glossary_control_flow_script]]
+Control Flow Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run inside the forked Java Virtual Machine, after the <<_glossary_task_executable,*Task Executable*>>, to determine control flow actions.
+
+[[_glossary_clean_script]]
+Cleaning Script::
+  A script part of a <<_glossary_task,*ProActive Task*>> definition and run after the <<_glossary_task_executable,*Task Executable*>> and before releasing the <<_glossary_proactive_node,*ProActive Node*>> to the <<_glossary_resource_manager,*Resource Manager*>>.
+
+[[_glossary_script_bindings]]
+Script Bindings::
+   Named objects which can be used inside a <<_glossary_task_executable_script,*Script Task*>> or inside <<_glossary_additional_task_scripts,*Additional Task Scripts*>> and automatically defined by the <<_glossary_proactive_scheduler,*ProActive Scheduler*>>. The type of each script binding depends on the script language used.
```

Comment:
We can change `and automatically defined` into `and which are automatically defined``for better readability.

Reply:
ok